Created a new migration for database

trudge-64 opened the pull request:
Deleted old migration from Data Model and created a new migration

ccg-oharan commented:
Hope you don't mind, but I added something to your branch. I added a unit test that will try and run the migrations on an sqlite in memory database. This should help us catch future errors since I can't review the migrations in a pull request.

trudge-64 commented:
I don’t mind at all. Prefer it that way. I wouldn’t want to break the database and have the same issues again. :)

 

Theresa

 

From: ccg-oharan <notifications@github.com> 
Sent: December 1, 2020 11:00 AM
To: dfo-mpo/ccg-cct <ccg-cct@noreply.github.com>
Cc: trudge-64 <trudge@sympatico.ca>; Author <author@noreply.github.com>
Subject: Re: [dfo-mpo/ccg-cct] Created a new migration for database (#24)

 

Hope you don't mind, but I added something to your branch. I added a unit test that will try and run the migrations on an sqlite in memory database. This should help us catch future errors since I can't review the migrations in a pull request.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub <https://github.com/dfo-mpo/ccg-cct/pull/24#issuecomment-736645637> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/AK5EA3MMQWURHI75KQK7VQLSSUHHFANCNFSM4UIIBLNA> .  <https://github.com/notifications/beacon/AK5EA3N2OK2BC5FPS5FVP7LSSUHHFA5CNFSM4UIIBLNKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOFPUFEBI.gif>
